pilot/pkg/model/authorization.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+const (
+	// RbacConfigName is the name of the RbacConfig custom resource that controls the RBAC behavior.
+	RbacConfigName = "rbac-config"
+
+	// RbacConfigNamespace is the namespace of the RbacConfig custom resource that controls the RBAC behavior.
+	RbacConfigNamespace = "istio-system"
```

Comment:
Customer can chose to install Istio on different namespace. I'm not sure using hard-coded value is a good idea.
Also, do you consider to use cluster CRD instead, for global/mesh config. We did it for authn recently, and framework is more ready for you to do so now.

Reply:
Updated to cluster scope and also removed this hard code.